features/step_definitions/user_profile_form_steps.rb

```diff
@@ -0,0 +1,133 @@
+When(/^I should see my user info$/) do
+  user = UserPresenter.new(@user)
+
+  expect(page).to have_text(user.duns_number)
+  expect(page).to have_text(user.email)
+  expect(page).to have_text(user.name)
+  expect(page).to have_text(user.github_id)
+  expect(page).to have_text(user.in_sam?)
+end
+
+Then(/^I should see a profile form$/) do
+  expect(page).to have_content("Complete your account")
+  expect(page).to have_content("Name")
+  expect(page).to have_content("DUNS Number")
+  expect(page).to have_content("Email Address")
+end
+
+Then(/^I should see a profile form with my info$/) do
+  expect(page).to have_content("Complete your account")
+
+  field = find_field('Name')
+  expect(field.value).to eq(@user.name)
+  field = find_field("DUNS Number")
+  expect(field.value).to eq(@user.duns_number)
+  field = find_field("Email Address")
+  expect(field.value).to eq(@user.email)
+end
+
+When(/^I fill out the profile form$/) do
+  @new_name = Faker::Name.name
+  @new_duns = Faker::Company.duns_number
+  @new_email = "random#{rand(10000)}@example.com"
+
+  fill_in("user_name", with: @new_name)
+  fill_in("user_duns_number", with: @new_duns)
+  fill_in("user_email", with: @new_email)
+end
+
+When(/^I fill the "([^"]*)" field with "([^"]*)"$/) do |field, value|
+  fill_in(field, with: value)
+end
+
+When(/^there is no (.+) associated with my account$/) do |attribute|
+  attribute = attribute.parameterize('_')
+  @user.update_attribute(attribute, nil)
+end
+
+Then(/^my (.+) should not be set$/) do |attribute|
+  attribute = attribute.parameterize('_')
+  expect(@user.send(attribute)).to be_blank
+end
+
+def fake_value_for_attribute(attribute)
+  case attribute
+  when 'credit_card_form_url'
+    Faker::Internet.url.gsub('http:', 'https:')
+  when 'name'
+    Faker::Name.name
+  when 'duns_number'
+    Faker::Company.duns_number
+  when 'email'
+    "random#{rand(10000)}@example.com"
+  else
+    fail "Unknown attribute '#{attribute}'"
+  end
+end
+
+When(/^there is a (.+) associated with my account$/) do |attr|
```

Comment:
oh it looks like we *do* have a file like that! Tis step and the one above seem like they should do there?